snowflake/connect.go

```diff
@@ -0,0 +1,277 @@
+package snowflake
+
+import (
+	"context"
+	"crypto/rsa"
+	"database/sql"
+	"encoding/json"
+	"encoding/pem"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+
+	homedir "github.com/mitchellh/go-homedir"
+	"github.com/pkg/errors"
+	gosnowflake "github.com/snowflakedb/gosnowflake"
+	"github.com/youmark/pkcs8"
+	"golang.org/x/crypto/ssh"
+
+	"github.com/turbot/steampipe-plugin-sdk/v2/plugin"
+)
+
+func connect(ctx context.Context, d *plugin.QueryData) (*sql.DB, error) {
```

Comment:
Does the TF provider or SDK accept env variables? If so, do we accept them here as well?